scala-package/examples/pom.xml

```diff
@@ -147,5 +174,12 @@
       <artifactId>opencv</artifactId>
       <version>2.4.9-7</version>
     </dependency>
+    <!-- https://mvnrepository.com/artifact/org.mockito/mockito-all -->
```

Comment:
do you need mockito for testing the example. this should be like a integration test